atd-toolbox/direction_movement/docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+# the usual cycle: docker-compose stop; docker-compose build --no-cache; docker-compose up -d; docker ps; docker exec -it vz_mvt_dir_recovery_environment bash
+
+version: '3.7'
+services:
+  recovery_environment:
+    container_name: vz_mvt_dir_recovery_environment
+    build: recovery_environment
+    environment:
+      POSTGRES_PASSWORD: moped
+      POSTGRES_USER: moped
+      POSTGRES_DB: moped
+      POSTGRES_HOST_AUTH_METHOD: trust
+      PGDATA: /var/lib/postgresql/data/pgdata
```

Comment:
This works fine, but it's obviously drawn from moped. I'll drop some more VZ oriented credentials in here. However, it's worth noting that anything works here, it's just confusing to have another, unrelated project's name used.